source/common/http/mixed_conn_pool.cc

```diff
@@ -50,6 +50,7 @@ void HttpConnPoolImplMixed::onConnected(Envoy::ConnectionPool::ActiveClient& cli
 
   Upstream::Host::CreateConnectionData data{std::move(tcp_client->connection_),
                                             client.real_host_description_};
+  data.connection_->readDisable(false);
```

Comment:
Why is this needed? I haven't followed development of mixed_conn_pool; can this take a connection from the tcp-proxy pool which was read-disabled?

Reply:
yeah, the ALPN pool creates a raw TCP connection, then slaps on an H1/H2 codec once we have TLS handshake results.  I'll add a comment to clarify